src/storage/util.rs

```diff
@@ -0,0 +1,64 @@
+use serde::{Serialize, Serializer};
+
+type ChronoUtc = chrono::DateTime<chrono::Utc>;
+
+/// A wrapper around DateTime<Utc> from chrono that allows us
+/// to implement Serialize (otherwise, Rust forbids implementing
+/// a foreign trait for a foreign type from a different crate).
```

Comment:
Rust does not have a built-in type for date type?

Reply:
Nope :( chrono seems to be the most popular one.

Reply:
What a shame

Reply:
What a shame.